csplit/src/csplit.yml

```diff
@@ -0,0 +1,64 @@
+name: csplit
+version: "0.1.0"
+author: Mats Kindahl <mats.kindahl@gmail.com>
+about: |-
+  Split FILE by each PATTERN into a series of files 'xx00', 'xx01',
+  ... 
+
+  A file name has to be provided, but if it is '-', input will read
+  from standard input.
+
+  The input will be split up to, but not including, the line that
+  matches PATTERN.
+
+  The number of bytes in each file will be displayed on standard
+  output with one line for each file.
+after_help: |-
+  PATTERN can be any of:
+    INTEGER            copy lines up to line number
+    /REGEXP/[OFFSET]   copy lines up to line matching REGEXP
+    %REGEXP%[OFFSET]   skip lines up to line matching REGEXP
+    {INTEGER}          repeat preceeding pattern INTEGER times
+    {*}                repeat preceeding pattern indefinitely
+
+  If an OFFSET is given it should be an integer, either positive or
+  negative. An offset without sign is assumed to be positive.
+args:
+    - FILE:
+        help: File to read, or '-' to read from standard input.
+        required: true
+    - PATTERN:
+        help: Patterns to use when splitting file.
+        multiple: true
+    - prefix:
+        long: prefix
+        short: f
+        value_name: PREFIX
+        help: Prefix to use for written files. The default is 'xx'.
+    - suffix-format:
+        long: suffix-format
+        short: b
+        value_name: FORMAT
+        help: Format to use for the file suffix. The default is '%02d'.
```

Comment:
You can add a default field here with the value. It will also be shown in the help

Reply:
Done.